optaplanner-benchmarks/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kie-benchmarks</artifactId>
+    <groupId>org.kie</groupId>
+    <version>7.8.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie</groupId>
+  <artifactId>optaplanner-benchmarks</artifactId>
+  <packaging>pom</packaging>
+  <name>OptaPlanner Benchmarks Parent</name>
+  <description>This is the multi-module project top-level parent.</description>
+
+  <modules>
+    <module>optaplanner-perf-framework</module>
+    <module>optaplanner-perf-benchmark</module>
+  </modules>
+
+  <properties>
+    <version.optaplanner>7.8.0-SNAPSHOT</version.optaplanner>
+    <version.jmh>1.20</version.jmh>
```

Comment:
There is a <version.jmh> property in root pom.xml. If the version needs to be raised please do so.

Reply:
I know, question is - is it safe to just raise the JMH version for all of us without running benchmarks and comparing with results before the change?

I would prefer raising JMH version in a separate PR so that's clearly visible to all of us.

Reply:
This is master branch. We have 7.7.x and 7.5.x for "stable" things.